### PR TITLE
[79] add project name to user table

### PR DESCRIPTION
**Pull request by caravinci**
new column into a table, to help identify users

### CODE REVIEW

File: enrolment-ui/public/locales/en/translation.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     },
     "RequestTable": {
         "userTableHeader":
-            "<0>Request ID</0><1>Name / Email</1><2>Created Date</2><3>Updated Date</3><4>Status</4><5>Action</5>",
+            "<0>Request ID</0><1>Name / Email</1><2>Project Name</2><3>Created Date</3><4>Updated Date</4><5>Status</5><6>Action</6>",
```
Comment (caravinci) [inline enrolment-ui/public/locales/en/translation.json:43]: new field label, in English

File: enrolment-ui/public/locales/fr/translation.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
     },
     "RequestTable": {
         "userTableHeader":
-            "<0>RID de la demande</0><1>Nom/Email</1><2>Date de création</2><3>Date de mise à jour</3><4>État</4><5>Action</5>",
+            "<0>RID de la demande</0><1>Nom/Email</1><2>Projet</2><3>Date de création</3><4>Date de mise à jour</4><5>État</5><6>Action</6>",
```
Comment (caravinci) [inline enrolment-ui/public/locales/fr/translation.json:43]: new field label, in Français

File: enrolment-ui/src/modules/ProjectUsers/UserRequests.js [hunk 3]
```diff
@@ -63,6 +65,8 @@ const UserRequests = ({
                             }
                         </td>
 
+                        <td>{getProjectNameById(user.project) || '--'}</td>
```
Comment (caravinci) [inline enrolment-ui/src/modules/ProjectUsers/UserRequests.js:68]: show value if available

File: enrolment-ui/src/modules/ProjectUsers/UserRequests.js [hunk 4]
```diff
@@ -130,4 +135,11 @@ const mapDispatchToProps = dispatch => ({
     resendInviteToProjectUser(dispatch, projectId, id, next),
 });
 
-export default withTranslation()(connect(mapStateToProps, mapDispatchToProps)(UserRequests));
+const mergeProps = ({ projects, ...stateProps }, dispatchProps, ownProps) => ({
+    ...ownProps,
+    ...dispatchProps,
+    ...stateProps,
+    getProjectNameById: projectID => (projects.find(({ id }) => projectID === id) || {}).project_name
```
Comment (caravinci) [inline enrolment-ui/src/modules/ProjectUsers/UserRequests.js:142]: This gets the project name from the projects list

**Review by caravinci — COMMENTED** [review]
Comments to help reviewing

**Review by ciaranschutte — APPROVED** [review]

**Pull request merged**